File: 1010 - Simple Calculate/Main.js
let inputs = require("fs").readFileSync("/dev/stdin", "utf8").split("\n");
let [, units1, ppu1] = inputs[0].split(" ");
let [, units2, ppu2] = inputs[1].split(" ");
console.log(
  `VALOR A PAGAR: R$ ${(
    parseInt(units1) * parseFloat(ppu1) +
    parseInt(units2) * parseFloat(ppu2)
  ).toFixed(2)}`
);

File: 1010 - Simple Calculate/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        scan.nextInt();
        int units1 = scan.nextInt();
        double ppu1 = scan.nextDouble();
        scan.nextInt();
        int units2 = scan.nextInt();
        double ppu2 = scan.nextDouble();
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", units1 * ppu1 + units2 * ppu2);
        scan.close();
    }
}